examples/shard-worker.config.example

```diff
@@ -5,6 +5,11 @@ port: 8981
 # by all shard cluster servers
 public_name: "localhost:8981"
 
+# This determines whether the shard instance will participate in the backplane
+# and cache it's operation outputs in its own cache file CAS.
+# When not caching locally, the worker will send its output to another worker's CAS.
```

Comment:
prefer 'whether the worker is a shard of the cas. When not a shard, it will upload into the Sharded CAS'